core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -312,4 +326,215 @@ private MetricsReporter metricsReporter() {
 
     return metricsReporter;
   }
+
+  protected ViewOperations newViewOps(TableIdentifier viewIdentifier) {
+    throw new UnsupportedOperationException("Not implemented: newViewOps");
+  }
+
+  @Override
+  public List<TableIdentifier> listViews(Namespace namespace) {
+    throw new UnsupportedOperationException("Not implemented: listViews");
+  }
+
+  @Override
+  public View loadView(TableIdentifier identifier) {
+    throw new UnsupportedOperationException("Not implemented: loadView");
```

Comment:
The purpose of `newTableOps` is to be able to define `loadTable` in this class. I think this should either not use `ViewOperations` or should implement `loadView` in a similar way.

Reply:
makes sense, I've updated `loadView()` but omitted the metadata handling that `loadTable()` is doing